Tests/TestCase.php

```diff
@@ -15,6 +15,61 @@
 
 class TestCase extends BaseTestCase
 {
+    public function createDbalOnlyTestContainer(): ContainerBuilder
+    {
+        $container = new ContainerBuilder(new ParameterBag([
+            'kernel.name' => 'app',
```

Comment:
Are you replicating what FrameworkBundle is doing here? This is prone to breakages in my experience. Why not having DBALTestKernel, similarly as we have [TestKernel](https://github.com/doctrine/DoctrineBundle/blob/ec69ba812cbf768c0fdabfc1cff663d14ae7109c/Tests/DependencyInjection/Fixtures/TestKernel.php)? Or even just injection of argument via constructor of TestKernel which enables/disables orm.

Reply:
I must admit I copy pasted a lot of what I found in `createXmlBundleTestContainer` here… maybe I can reduce this to something more simple.

Reply:
Addressed, it's indeed better like this I think :)